index.html

```diff
@@ -33,14 +33,29 @@
       <li>Design: Jewelry, Interiors, Architecture, Fashion Styling, Costume Design, &amp; Graphics</li>
       <li>Movement: Yoga, Qi Gong, Urban Walking, &amp; Hiking</li>
     </ul>
+    <h2>Top Ten Countries I Want to Visit</p>
+    <ol>
+        <li>Thailand</li>
+        <li>Bali</li>
+        <li>Costa Rica</li>
+        <li>The Seychelle Islands</li>
+        <li>India</li>
+        <li>Iceland</li>
+        <li>Denmark</li>
+        <li>Sweden</li>
+        <li>France</li>
+        <li>Canada</li>
+    </ol>
     <h2>Guessing Game</h2>
     <p>In which state was I born?</p>
     <p>Do I prefer dogs over cats?</p>
     <p>Am I a morning person?</p>
     <p>What is my favorite holiday?</p>
     <p>Is purple my favorite color?</p>
-    <!-- <p>What are my favorite forms of exercise?</p>
-    <p>What are my favorite beverages?</p> -->
+    <p>What are my favorite forms of exercise?</p>
+    <p>What are my favorite beverages?</p>
+    <p>How many years have I been vegan?</p>
+    <p></p>
   </section>
   <script src ="./app.js"></script>
```

Comment:
If the script is in the same level of the directory, it SHOULD work okay without the ./ before the filename. It's good to be explicit, though.